### ISSUE TITLE
Start minimised in tray does not work in Joplin Portable in Windows

### ISSUE THREAD

**Issue by tomasz1986** [laurent22/joplin#2297]
I believe that this may the same problem as described in https://github.com/laurent22/joplin/issues/2077.

In one word, enabling the "Start application minimised in the tray icon" option in Joplin settings does nothing when using the portable version in Windows.

I have tested both with my normal Joplin Portable installation, and also with a completely clean installation. There is no difference between the two. The application always starts maximised, regardless of the setting.

## Environment

Joplin version: 1.0.175 (prod, win32)
Platform: Windows 7 Enterprise x64
OS specifcs: 

## Steps To Reproduce

1. Download and start JoplinPortable.exe
2. Enable "Start application minimised in the tray icon" in the Options.
3. Close the application completely and start JoplinPortable.exe again.

Describe what you expected to happen:

Joplin should start minimised in tray, without showing up in taskbar.

## Logfile

There are no errors both in the Console and in the log file.

**Comment by AddictArts** [comment]
Cannot reproduce on Windows 10 with the latest 1.0.177.

Be sure you choose File->Exit or go to the tray and right click select exit, my apologies if you had done this. Hope this helps.

**Comment by tomasz1986** [comment]
> Be sure you choose File->Exit or go to the tray and right click select exit, my apologies if you had done this. Hope this helps.

I have indeed done that, but now I have also tested it in two other systems - one with Windows 7 ThinPC (x86) and one with Windows 10 Enterprise 2019 LTSC (x64) - and the program does start minimised there, so I think that something may be going on with this specific Windows installation.

I will close, as I am not feeling like doing a full-fledge investigation, and the system is scheduled to be upgraded soon anyway. I am sorry for not checking more Windows installations before opening the issue.
